examples/step-44/CMakeLists.txt

```diff
@@ -34,6 +34,17 @@ IF(NOT ${deal.II_FOUND})
     )
 ENDIF()
 
+#
+# Are all dependencies fulfilled?
+#
+IF(NOT DEAL_II_WITH_CXX11)
+  MESSAGE(FATAL_ERROR "
+Error! The deal.II library found at ${DEAL_II_PATH} was not configured with
+    DEAL_II_WITH_CXX11 = ON
+One or all of these are OFF in your installation but are required for this tutorial step."
```

Comment:
Looking at the WorkStream code, no, it is not a requirement. It should work without TBB in a single-thread mode.